src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface that produces a printable string representation of an object.
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
can tell what this method is supposed to do.

Reply:
Does this mean I should comment on the method instead?